.gitignore

```diff
@@ -0,0 +1,131 @@
+# Logs
```

Comment:
There are too many things in this. Do you understand all of these lines? Please double-check and only use the necessary lines